.vscode/launch.json

```diff
@@ -63,9 +63,9 @@
       "name": "format format.sln --fix-analyzers warn --check",
       "type": "coreclr",
       "request": "launch",
-      "preLaunchTask": "publish",
+      "preLaunchTask": "build",
       // If you have changed target frameworks, make sure to update the program path.
-      "program": "${workspaceFolder}/artifacts/bin/dotnet-format/Debug/netcoreapp2.1/publish/dotnet-format.dll",
+      "program": "${workspaceFolder}/artifacts/bin/dotnet-format/Debug/netcoreapp2.1/dotnet-format.dll",
```

Comment:
Updated the dotnet-format project to copy package assemblies into build folder. This is needed for finding the Feature assemblies.